### PR TITLE
Fix an issue where PYTHONPATH and callback_dir can't be set together

### PR DESCRIPTION
**Pull request by matburt**
Related: #132

### CODE REVIEW

File: ansible_runner/runner_config.py [hunk 1]
```diff
@@ -128,7 +131,8 @@ def prepare(self):
         self.env['ANSIBLE_RETRY_FILES_ENABLED'] = 'False'
         self.env['ANSIBLE_HOST_KEY_CHECKING'] = 'False'
         self.env['AWX_ISOLATED_DATA_DIR'] = self.artifact_dir
-        self.env['PYTHONPATH'] = self.env.get('PYTHONPATH', '') + callback_dir + ':'
+
+        self.env['PYTHONPATH'] = python_path + callback_dir + ':'
```
Comment (benthomasson) [inline ansible_runner/runner_config.py]: How about?
```
self.env['PYTHONPATH'] = ":".join(filter(None, [python_path, callback_dir]))
```

File: test/unit/test_runner_config.py [hunk 2]
```diff
@@ -300,6 +300,10 @@ def test_prepare():
     assert rc.env['AWX_ISOLATED_DATA_DIR'] == '/'
     assert rc.env['PYTHONPATH'] == '/:'
 
+    os.environ['PYTHONPATH'] = "/foo/bar"
+    rc.prepare()
+    assert rc.env['PYTHONPATH'] == "/foo/bar:/:"
```
Comment (benthomasson) [inline test/unit/test_runner_config.py]: Is the trailing `:` required?

File: ansible_runner/runner_config.py (hunk 1 above)
Comment (matburt) [reply ansible_runner/runner_config.py]: meh? It's just two static elements

File: test/unit/test_runner_config.py (hunk 2 above)
Comment (matburt) [reply test/unit/test_runner_config.py]: it's not *not* required, and can be helpful during future concatenations.

**Comment by benthomasson** [comment]
mergeit

**Pull request merged**